core/src/main/java/org/apache/calcite/rel/rules/IntersectToDistinctRule.java

```diff
@@ -44,25 +48,16 @@
  *
  * <h2>Example</h2>
  *
- * <p>Query: <code>R1 Intersect All R2</code>
+ * <p>Query: <code>R1 Intersect R2</code>
```

Comment:
Can you revise this whole section? The previous author used terms such as 'GB' (which is used in Hive but not in Calcite). I suspect that SQL queries are clearer than pseudo-relational algebra.